### PR TITLE
Refactor the Calculatator.js file into function component

### PR DESCRIPTION
**Pull request by The-Normal-One1**
### these were done to refactor:

- Changed lass component into function component.
- Imported `useState` from React to my code and remove the component.

### CODE REVIEW

File: src/components/Calculator.js [hunk 1]
```diff
@@ -1,190 +1,132 @@
...
-    );
-  }
+  const results = `
+  ${
+  data.total || data.operation || data.next
+    ? `${data.total || ''} ${data.operation || ''} ${data.next || ''}`
+    : 0
+}`;
+
+  // data.next || data.total || 0
```
Comment (Davidosky007) [inline src/components/Calculator.js:24]: * Kindly remove all commented code from your code base as it makes your code base bulky and unprofessional 🙏

**Review by Davidosky007 — CHANGES_REQUESTED** [review]
Hi @The-Normal-One1 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
## Highlights
* No linter errors 👍 
* Well written PR description 👍 
* Proper use of Gitflow 👍 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @davidsoky007 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by frank1738 — COMMENTED** [review]
Hi  @The-Normal-One1 , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project with failing linter errors. Kindly fix them and resubmit your project

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Review by frank1738 — APPROVED** [review]
Hi @The-Normal-One1 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights

✅ Correct use of Gitflow
✅ No linter error
✅ README is okay

## Optional suggestions

N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**